### ISSUE TITLE
comment-format TSX false positive

### ISSUE THREAD

**Issue by Avol-V** [palantir/tslint#2007]
### Bug Report

- __TSLint version__: `4.3.1`
- __TypeScript version__: `2.1.4`
- __Running TSLint via__: CLI / Visual Studio Code

#### TypeScript code being linted

```tsx
const a = (
	<a href="https://github.com/">
		https://github.com/
	</a>
);
```

with `tslint.json` configuration:

```json
{
	"extends": "tslint:recommended"
}
```

#### Actual behavior

```
test.tsx[3, 11]: comment must start with a space
```

#### Expected behavior

Should be OK.
